building/README.md

```diff
@@ -21,8 +21,12 @@ This chapter contains instructions on how to build a reference project and how t
 
 Instructions in the `Building` and `Running system on targets` chapters have been verified for the Ubuntu
 (20.04 and 22.04 versions) Linux distribution and macOS (tested on macOS Monterey 12.6.1), so this is the easiest way
-to start working with Phoenix-RTOS. Using Windows isn't supported yet, but you can create a virtual machine with Ubuntu
-or try to use Docker.
+to start working with Phoenix-RTOS. Using Windows is possible by using tools like `Cygwin` or `WSL`. The other way is to
+try to use Docker.
```

Comment:
I think that we can resign from suggesting Docker as we don't have it described and using wsl is much simpler.

```suggestion
to start working with Phoenix-RTOS. Windows is also supported, by using `Cygwin` or `WSL`.
```